tests/unit/amo/components/TestAddonReviewListItem.js

```diff
@@ -442,4 +460,88 @@ describe(__filename, () => {
 
     expect(root.find(ErrorList)).toHaveLength(1);
   });
+
+  describe('Developer reply to a review', () => {
+    const _setReviewReply = ({ addon = fakeAddon } = {}) => {
+      const review = _setReview({
+        ...fakeReview,
+        addon: {
+          id: addon.id,
+          slug: addon.slug,
+        },
+        id: 1,
+        body: 'The original review',
+        reply: {
+          ...fakeReview,
+          id: 2,
+          body: 'Reply to the review',
+        },
+      });
+
+      return { review, reply: review.reply };
+    };
+
+    const renderReply = ({
+      addon = createInternalAddon(fakeAddon),
+      reply = _setReviewReply({ addon }).reply,
+      ...props
+    } = {}) => {
+      return render({
+        addon,
+        review: reply,
+        isReply: true,
+        ...props,
+      });
+    };
+
+    it('sets a reply class name', () => {
+      const root = renderReply();
+
+      expect(root).toHaveClassName('AddonReviewListItem-reply');
+    });
+
+    it('renders a nested reply', () => {
+      const addon = createInternalAddon(fakeAddon);
+      const { review, reply } = _setReviewReply({ addon });
+      const root = render({ addon, review });
+
+      const replyComponent = root.find(AddonReviewListItem);
+      expect(replyComponent).toHaveLength(1);
+      expect(replyComponent).toHaveProp('addon', addon);
+      expect(replyComponent).toHaveProp('review', reply);
+      expect(replyComponent).toHaveProp('isReply', true);
+    });
+
+    it('hides ratings for replies', () => {
+      const root = renderReply();
+
+      expect(root.find(Rating)).toHaveLength(0);
+    });
+
+    it('does not include a user name in the by line', () => {
+      const { reply } = _setReviewReply();
+      const root = renderReply({ reply });
+
+      expect(root.find('.AddonReviewListItem-by-line'))
+        .not.toIncludeText(reply.userName);
+    });
+
+    it('hides the edit button for replies', () => {
```

Comment:
Can developers not edit their replies? Huh. Seems like it would be useful.

Reply:
Oh I see, waiting on https://github.com/mozilla/addons-frontend/issues/3368. A TODO here about it might be handy.

Reply:
The TODO with a link to the issue in in the code. I think that's a better place to put TODOs. When you search for the TODO and remove the code, the test will fail so you will know which tests to delete. However, if you search for a TODO that leads to a test and you delete the test, you might not know which code needs updating.